openlibrary/catalog/get_ia.py

```diff
@@ -1,17 +1,22 @@
 from __future__ import print_function
-from openlibrary.catalog.marc.marc_binary import MarcBinary
-from openlibrary.catalog.marc.marc_xml import MarcXml
-from openlibrary.catalog.marc import fast_parse, parse
-from infogami import config
-from lxml import etree
-import xml.parsers.expat
-import urllib2
+
 import os.path
 import socket
-from time import sleep
 import traceback
+import xml.parsers.expat
+import urllib2
+
+from infogami import config
+from lxml import etree
+from time import sleep
+
+from openlibrary.catalog.marc.marc_binary import MarcBinary
+from openlibrary.catalog.marc.marc_xml import MarcXml
+from openlibrary.catalog.marc.parse import read_edition  # Deprecated import
```

Comment:
this import should not be deprecated!, `openlibrary.catalog.marc.parse.read_edition(MarcXML|MarcBinary)` is the preferred way to  parse a MARC record.

This was not obvious in the code, hence my confusion.
`fast_parse` should be deprecated completely. There is still valuable code in `parse`, but the structure may need some work and further refactoring.